README.md

```diff
@@ -7,8 +7,17 @@
 ## Installieren von SDL2
 
 Wir brauchen die Pakete sdl2 und sdl2_image.
+
 Installation auf Linux: distributionsspezifisch, sollte aber schon installiert sein
 
+Installation auf Windows:
+
+- Herunterladen der **32 Bit** Binaries von der [offiziellen Seite](https://www.libsdl.org/download-2.0.php)
+- die .tar.gz mit einer Archivsoftware öffnen (ich empfehle [7-zip](https://7-zip.org)
+- SDL2 entweder in die MinGW Ordnerstruktur integrieren ODER irgendwohin entpacken und Pfad merken
+
+Sollte SDL2 irgendwo hin entpackt worden sein, funktioniert ``FindPkgConfig`` eventuell nicht mehr
```

Comment:
Ich mal versucht SDL2 in die entsprechenden Ordner von MinGW zu entpacken, konnte das Ganze aber so nicht zum laufen bekommen. Blöder Weise habe ich auch nicht unbedingt die Geduld um mich die so lange mit CMake rumzuquälen. Mein Lösungsweg über den Pfad ist zwar nicht optimal, aber solange es für mich so funktioniert, reicht mir das😅  ``FindPkgConfig`` funktioniert für mich unter Windows weiterhin nicht.